src/Components/ChatContainer/ChatContainer.js

```diff
@@ -26,28 +25,26 @@ function ChatContainer({ groupCode, roomCode }) {
   };
 
   useEffect(() => {
-    socket.on("message",function(msg) {  
-      getMessages(msg); 
+    socket.on("message",function(msg, fun) {  
+      getMessages(msg);
+      console.log(fun);;
```

Comment:
why this instead of above?

Reply:
This was an old change and the only difference is fun where I was playing around with what could comback